client/my-sites/theme/main.jsx

```diff
@@ -243,7 +243,7 @@ class ThemeSheet extends React.Component {
 		return (
 			<div className="theme__sheet-preview-link">
 				<span className="theme__sheet-preview-link-text">
-					{ i18n.translate( 'Open Live Demo', {
+					{ i18n.translate( 'Open live demo', {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.